src/main/java/com/conveyal/taui/persistence/Persistence.java

```diff
@@ -57,14 +59,15 @@ public static void initialize () {
         regionalAnalyses = getTable("regional-analyses", RegionalAnalysis.class);
         bookmarks = getTable("bookmarks", Bookmark.class);
         aggregationAreas = getTable("aggregationAreas", AggregationArea.class);
+        opportunityDatasets = getTable("opportunityDatasets", OpportunityDataset.class);
     }
 
     /** Connect to a Mongo table using MongoJack, which persists Java objects into Mongo. */
     private static <V extends Model> MongoMap<V> getTable (String name, Class clazz) {
         DBCollection collection = db.getCollection(name);
         ObjectMapper om = JsonUtil.getObjectMapper(JsonViews.Db.class, true);
         JacksonDBCollection<V, String> coll = JacksonDBCollection.wrap(collection, clazz, String.class, om);
-        return new MongoMap<>(coll);
+        return new MongoMap<>(coll, clazz);
```

Comment:
Gives rise to a new theory on manzanas.  Maybe mansana was a reserved word...

Reply:
😆